exchanges/zb/zb_wrapper.go

```diff
@@ -379,32 +376,34 @@ func (z *ZB) SubmitOrder(order *exchange.OrderSubmission) (exchange.SubmitOrderR
 
 // ModifyOrder will allow of changing orderbook placement and limit to
 // market conversion
-func (z *ZB) ModifyOrder(action *exchange.ModifyOrder) (string, error) {
+func (z *ZB) ModifyOrder(action *order.Modify) (string, error) {
 	return "", common.ErrFunctionNotSupported
 }
 
 // CancelOrder cancels an order by its corresponding ID number
-func (z *ZB) CancelOrder(order *exchange.OrderCancellation) error {
-	orderIDInt, err := strconv.ParseInt(order.OrderID, 10, 64)
-
+func (z *ZB) CancelOrder(o *order.Cancellation) error {
+	orderIDInt, err := strconv.ParseInt(o.OrderID, 10, 64)
 	if err != nil {
 		return err
 	}
-
-	return z.CancelExistingOrder(orderIDInt, z.FormatExchangeCurrency(order.CurrencyPair,
-		order.AssetType).String())
+	curr := z.FormatExchangeCurrency(o.CurrencyPair, o.AssetType).String()
+	return z.CancelExistingOrder(orderIDInt, curr)
 }
 
 // CancelAllOrders cancels all orders associated with a currency pair
-func (z *ZB) CancelAllOrders(_ *exchange.OrderCancellation) (exchange.CancelAllOrdersResponse, error) {
-	cancelAllOrdersResponse := exchange.CancelAllOrdersResponse{
-		OrderStatus: make(map[string]string),
+func (z *ZB) CancelAllOrders(_ *order.Cancellation) (order.CancelAllResponse, error) {
+	cancelAllOrdersResponse := order.CancelAllResponse{
+		Status: make(map[string]string),
 	}
 	var allOpenOrders []Order
-	for _, currency := range z.GetEnabledPairs(asset.Spot) {
+	enabledPairs := z.GetEnabledPairs(asset.Spot)
+	for x := range enabledPairs {
 		// Limiting to 10 pages
-		for i := 0; i < 10; i++ {
-			openOrders, err := z.GetUnfinishedOrdersIgnoreTradeType(z.FormatExchangeCurrency(currency, asset.Spot).String(), 1, 10)
+		for pageNumber := int64(0); pageNumber < 11; pageNumber++ {
```

Comment:
Now 11 pages! 🦀 I know you've changed this to be way better in your other PR thought, so I'll leave this for now

Reply:
:man_shrugging: Who knows what I was doing there. Yeah will be updated in other QA. But I would rather revert it now to be honest. Good spotting.

Reply:
too hard basket.